backend/.env

```
@@ -0,0 +1,8 @@
+PORT=5000
+NODE_ENV=development   # change to 'production' in production
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The NODE_ENV key should go before the PORT key

(UnorderedKey)